File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Resources/setresourceproperties.rb
module Asposetasksjava
  module SetResourceProperties
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new

		# Resource properties are represented by static class Rsc
		rsc = project.getResources().add("Rsc")
		
		# set resource properties
		rsc.set(Rjb::import('com.aspose.tasks.Rsc').STANDARD_RATE, Rjb::import('java.math.BigDecimal').valueOf(15))
		rsc.set(Rjb::import('com.aspose.tasks.Rsc').OVERTIME_RATE, Rjb::import('java.math.BigDecimal').valueOf(20))

		puts "Set resource properties"
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithResources/SetResourceProperties.php
<?php
namespace Aspose\Tasks\WorkingWithResources;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Rsc as Rsc;

use java\math\BigDecimal as BigDecimal;

class SetResourceProperties{

    public static function run($dataDir=null){

        $project = new Project();

        # Resource properties are represented by static class Rsc
        $rsc = $project->getResources()->add("Rsc");

		# set resource properties
        $rscc=new Rsc();
        $bigDecimal=new BigDecimal();
		$rsc->set($rscc->STANDARD_RATE, $bigDecimal->valueOf(15));
		$rsc->set($rscc->OVERTIME_RATE, $bigDecimal->valueOf(20));

		print "Set resource properties".PHP_EOL;

    }

}
?>